pandas/core/strings.py

```diff
@@ -898,23 +898,92 @@ def str_join(arr, sep):
 
 def str_findall(arr, pat, flags=0):
     """
-    Find all occurrences of pattern or regular expression in the
-    Series/Index. Equivalent to :func:`re.findall`.
+    Find all occurrences of pattern or regular expression in the Series/Index.
+
+    Equivalent to applying :func:`re.findall` to all the elements in the
+    Series/Index.
 
     Parameters
     ----------
     pat : string
-        Pattern or regular expression
-    flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
+        Pattern or regular expression.
+    flags : int
```

Comment:
Can you make this `int, default 0` (we changed our minds about this, the docstring guide has been updated, sorry about that)

Reply:
Note that the docstring guide indicates: `int (default 0)`

Reply:
yes, but as I said we decided to change that (and I think now it should be reflected in the online guide, although it might take another rebuild to reflect it)